app/code/Magento/LayeredNavigation/view/frontend/templates/layer/filter.phtml

```diff
@@ -7,35 +7,35 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
 <?php
-/**
- * Template for filter items block
- *
- * @var $block \Magento\LayeredNavigation\Block\Navigation\FilterRenderer
- */
+/** @var $block \Magento\LayeredNavigation\Block\Navigation\FilterRenderer */
+/** @var \Magento\LayeredNavigation\ViewModel\Layer\Filter $viewModel */
+$viewModel = $block->getData('product_layer_view_model');
 ?>
 
 <ol class="items">
-    <?php foreach ($filterItems as $filterItem) : ?>
+    <?php foreach ($filterItems as $filterItem): ?>
         <li class="item">
-            <?php if ($filterItem->getCount() > 0) : ?>
-                <a href="<?= $block->escapeUrl($filterItem->getUrl()) ?>">
+            <?php if ($filterItem->getCount() > 0): ?>
+                <a href="<?= $block->escapeUrl($filterItem->getUrl()) ?>" rel="nofollow">
                     <?= /* @noEscape */ $filterItem->getLabel() ?>
-                    <?php if ($this->helper(\Magento\Catalog\Helper\Data::class)->shouldDisplayProductCountOnLayer()) : ?>
-                        <span class="count"><?= /* @noEscape */ (int)$filterItem->getCount() ?><span class="filter-count-label">
-                            <?php if ($filterItem->getCount() == 1) :
+                    <?php if ($viewModel->shouldDisplayProductCountOnLayer()): ?>
+                        <span class="count"><?= /* @noEscape */ (int)$filterItem->getCount() ?>
```

Comment:
when inside span tag adding new line or space - usually it adds visually additional space. Could you double check that we don't have regression here?

Reply:
ok